scripts/docker_build.sh

```diff
@@ -18,8 +18,12 @@
 # SPDX-License-Identifier: Apache-2.0
 # License-Filename: LICENSE
 
+# Docker from ORT uses BuildKit feature
+
 GIT_ROOT=$(git rev-parse --show-toplevel)
-GIT_REVISION=$(git describe --abbrev=10 --always --tags --dirty)
+GIT_VERSION=$(git describe --abbrev=10 --always --tags --dirty)
+ORT_DOCKER=${ORT_DOCKER:-ort}
+ARCH_TRANSLATION=""
```

Comment:
These changes (renaming the `GIT_REVISION` variable, making the Docker tag to use configurable, etc.) are unrelated to the topic described in the commit message.